content/api/vision.md

```diff
@@ -25,17 +25,17 @@ for marker in markers:
 ```
 
 ## Marker properties
-The markers that are contained within the array have some useful properties:
+The markers that are contained within the list have some useful properties:
 
   - `is_wall_marker()` - returns whether or not the marker is a wall marker.
   - `is_token_marker()` - returns whether or not the marker is a token marker.
   - `id` - returns the id of the marker.
   - `pixel_centre` - returns the pixel coordinates of the centre of the marker.
-  - `polar` - returns a `PolarCoord` object containing the position of the marker in the Polar coordinate system.
+  - `polar` - returns a `PolarCoord` object containing the position of the marker in the polar coordinate system.
   - `cartesian` - returns a `CartCoord` object containing the position of the marker in the Cartesian coordinate system.
 
-## Cartesian and Polar Coordinate Systems
-The markers' position can be represented using both the Polar and Cartesian coordinate system. The two objects `CartCoord` and `PolarCoord` hold the position information for both of these systems.
+## Cartesian and polar coordinate systems
+The markers' position can be represented using both the polar and Cartesian coordinate system. The two objects `CartCoord` and `PolarCoord` hold the position information for both of these systems.
```

Comment:
If _polar_ is lower cased here, shouldn't _Cartesian_?

Reply:
[_Cartesian_][c] is a proper noun (named after René Descartes), while [_polar_][p] isn't.

[c]: https://en.wikipedia.org/wiki/Cartesian_coordinate_system
[p]: https://en.wikipedia.org/wiki/Polar_coordinate_system